cmd/snap-confine/mount-opt.c

```diff
@@ -113,3 +117,122 @@ const char *sc_mount_opt2str(unsigned long flags)
 	}
 	return buf;
 }
+
+char *sc_mount_cmd(const char *source, const char *target,
+		   const char *filesystemtype, unsigned long mountflags,
+		   const void *data)
+{
+	char buf[100 + PATH_MAX * 2];
+	char *special = NULL;
+	int used_special_flags = 0;
+	strcpy(buf, "mount");
+
+	// Check for some special, dedicated syntax. This exists for:
+	// - bind mounts (bind)
+	//   - including the recursive variant
+	if (mountflags & MS_BIND) {
+		if (mountflags & MS_REC) {
+			special = " --rbind";
+		} else {
+			special = " --bind";
+		}
+		used_special_flags |= MS_BIND | MS_REC;
+	} else if (mountflags & MS_MOVE) {
```

Comment:
Do we need the `else if` here? If the user gives a combination that is not part of this if/else if/... tree (maybe by accident) we won't get accurate logging? Or is this no concern for some reason?